spec.bs

```diff
@@ -934,6 +983,30 @@ To <dfn>validate and convert auction ad config</dfn> given an {{AuctionAdConfig}
       1. Set |auctionConfig|'s [=auction config/seller signals=] to the result of
         [=serializing a JavaScript value to a JSON string=], given |result|.
     * To handle an error, set |auctionConfig|'s [=auction config/seller signals=] to failure.
+1. If |config|["{{AuctionAdConfig/auctionNonce}}"] [=map/exists=]:
+  1. Set |auctionConfig|'s [=auction config/auction nonce=] to the result of running
+    [=get uuid from string=] with |config|["{{AuctionAdConfig/auctionNonce}}"].
+1. If |config|["{{AuctionAdConfig/additionalBids}}"] [=map/exists=]:
+  1. [=exception/Throw=] a {{TypeError}} if any of the following conditions hold:
+    * |config|["{{AuctionAdConfig/auctionNonce}}"] does not [=map/exist=];
+    * |config|["{{AuctionAdConfig/interestGroupBuyers}}"] does not [=map/exist=], or it's [=list/empty=];
+    * |config|["{{AuctionAdConfig/componentAuctions}}"] is not [=list/empty=].
+  1. Set |auctionConfig|'s [=auction config/expects additional bids=] to true.
+  1. [=Handle an input promise in configuration=] given |auctionConfig| and
+    |config|["{{AuctionAdConfig/additionalBids}}"]:
+
+    Note: The JavaScript code calling {{Navigator/runAdAuction()}} is responsible for not resolving
+      |config|["{{AuctionAdConfig/additionalBids}}"] until a {{WindowOrWorkerGlobalScope/fetch()}}
+      call for fetching additional bids with the `{adAuctionHeaders: true}` option returns

```

Comment:
Let's remember to link to the proper `adAuctionHeaders` option/member, when that PR lands.